File: backend/src/main/java/com/incidentcommand/dto/UpdateIncidentRequest.java
package com.incidentcommand.dto;

import com.incidentcommand.model.enums.IncidentStatus;
import com.incidentcommand.model.enums.IncidentType;
import com.incidentcommand.model.enums.Severity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UpdateIncidentRequest {
    private String title;
    private String description;
    private Severity severity;
    private IncidentStatus status;
    private IncidentType type;
    private Long assignedTeamId;
    private Long assignedUserId;
}

File: android/app/src/main/java/com/incidentcommand/android/data/remote/dto/UpdateIncidentRequest.kt
package com.incidentcommand.android.data.remote.dto

data class UpdateIncidentRequest(
    val title: String? = null,
    val description: String? = null,
    val severity: String? = null,
    val status: String? = null,
    val type: String? = null,
    val assignedTeamId: Long? = null,
    val assignedUserId: Long? = null
)
